compute/rn/release-information/release-notes-22-12-update1.adoc

```diff
@@ -0,0 +1,93 @@
+:toc: macro
+== 22.12 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.12.xxx
+
+|Codename
+|Lagrange, 22.12 Update 1
+|Release date
+|Jan 9, 2023
```

Comment:
@Pubs-MV I think this is not the release date

Reply:
@grevach Thanks. Updated the tentative date for now.